src/assets/i18n/de.json

```diff
@@ -186,6 +186,9 @@
         "Masterbooks": "Meisterbücher",
         "Patreon_supporter": "Patreon-Unterstützer"
     },
+    "LISTS": {
+        "Merge_lists": "Merge lists"
```

Comment:
@Nivrelin You can tell me the translation on discord too, just need it for this line ^^